src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ProcessedEnvelopeMessageHandlingTest.java

```diff
@@ -58,10 +58,13 @@ public void should_complete_envelope_referenced_by_queue_message() throws Except
         UUID envelopeId = getEnvelope(zipFilename).getId();
 
         // when
-        String ccdId = "ccd-id";
-        String ccdAction = "ccd-action";
-        sendProcessedEnvelopeMessage(envelopeId, ccdId, ccdAction);
-
+        String ccdAction;
```

Comment:
can we use ternary here and send message with if statement?

Reply:
do not see too much point if there will be 'if'